src/components/chip-set/examples/chip-set-input-type-search.tsx

```diff
@@ -0,0 +1,81 @@
+import { Chip } from '@limetech/lime-elements';
```

Comment:
I squashed the commit with these examples into the feature commit, as I don't think they need to be separate, since it's docs for the feature you added.

I also added a fixup where I reworded some of the descriptions a little.

I've only tested this quickly on an iPhone on Browserstack, but didn't get any autocorrection there regardless of input type… 🤔

I'd like to test a little bit more, but I need to log off for today now.

Reply:
On Browserstack, I'm not getting autocorrection on either iOS or Android, regardless of whether I test the `text` or the `search` example.

I am getting the magnifying glass icon on the enter key on Android though. For the picker I guess that's good. For the chip-set example it seems quite misplaced though…

Using `search`:
<img width="350" alt="A screenshot of the limel-example-chip-set-input-type-search component with Android's software keyboard open, showing a magnifying glass icon on the Enter key" src="https://user-images.githubusercontent.com/154725/168546036-cb0f84d1-d7da-44f3-b756-d7efe0b49bc6.png">

Using `text`:
<img width="350" alt="A screenshot of the limel-example-chip-set-input-type-text component with Android's software keyboard open, showing an arrow-pointing-right icon on the Enter key" src="https://user-images.githubusercontent.com/154725/168545941-cccf6e91-a840-4c92-87d8-594cc6af01b7.png">

I do think we should allow setting the type regardless, so we can use `search` for the picker, but I think we might also want to allow some more granular control, like that described [here](https://davidwalsh.name/disable-autocorrect). That would be something we'd first enable in limel-input though…

Reply:
Ah, we don't use limel-input in the chips-set, right? So, no, we don't need to implement anything on limel-input before implementing it here 🙂

Reply:
> we don't use limel-input in the chips-set, right?

No, we don't use limel-input here.

Reply:
sorry for the delayed response. Yes the granular controls is a great idea. Actually both in here and in limel-input. :) But setting type to search sounds also like these granular controls. no?

> <img width="308" alt="image" src="https://user-images.githubusercontent.com/35954987/168973170-06af677e-33ba-413a-bd5b-75b42dea1015.png">

interesting effect here. I didn't know it works like that in Android.

> On Browserstack, I'm not getting autocorrection on either iOS or Android,

I think they have turned these features off on their keyboard settings. Probably they were problematic, given the service is used world-wide.

Reply:
> sorry for the delayed response. Yes the granular controls is a great idea. Actually both in here and in limel-input. :) But setting type to search sounds also like these granular controls. no?

> I do think we should allow setting the type regardless, so we can use `search` for the picker

😄

> I think they have turned these features off on their keyboard settings. Probably they were problematic, given the service is used world-wide.

Makes sense I guess… it's just a pity if it can't be turned on and off so it can be tested 😞 But I haven't looked, maybe there _is_ a setting for it 🤷‍♂️

Reply:
yes. When we tested on a real iOS device, the behavior changed as follows:

- The auto-correction would not automatically kick in before pressing space or enter. 
- the keyboard's predictive and spelling suggestions would still show word suggestions on keyboard's top bar. So user would notice that they could pick another word, or some spelling might be wrong. Like this image 👇 
![image](https://user-images.githubusercontent.com/35954987/169012493-2fbe196f-2653-46b7-889a-a8245aad7188.png)

I don't know how Android behaves though